src/V3UniqueNames.h

```diff
@@ -25,17 +25,27 @@
 #include "V3Hasher.h"
 
 #include <string>
-#include <unordered_map>
+#include <map>
 
 class V3UniqueNames final {
+public:
+    enum runMode {
```

Comment:
```suggestion
    enum runMode : uint8_t {
```

Though perhaps just take a bool as to if use a suffix.

Reply:
Added `uint8_t` type specifier. Using a enum may be easier to read and for future extending?

Reply:
Ok to leave the enum if you think it will be useful - but if there's no plans presently just seems extra complexity.